src/components/TodoSection.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+import TodoCard from './TodoCard';
+
+let TodoSectionLayout = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  flex: 1 1;
+  overflow-y: scroll;
+`;
+let TodoContentBox = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  flex: 1 1;
+  overflow-y: scroll;
+`;
+let TodoListHeaderBox = styled.div`
+  font-size: 0.75rem;
+  padding: 0.25rem 1rem;
+  font-weight: 700;
+  color: #9e9e9e;
+`;
+
+export default function TodoSection({ todoList, deleteTodo, toggleTodo }) {
+  return (
+    <TodoSectionLayout>
+      <TodoListHeaderBox>todo</TodoListHeaderBox>
+      <TodoContentBox>
+        {todoList.map((todo, index) => {
+          return (
+            <TodoCard
+              key={index}
+              todo={todo.todo}
+              isdone={todo.isdone}
+              deleteTodo={deleteTodo}
+              toggleTodo={toggleTodo}
+            />
+          );
+        })}
```

Comment:
map 함수 index까지 사용해서 너무 활용 잘하셨네요 😆

이렇게 배열 데이터를 순회하면서 컴포넌트에 **mapping**해주는 일을 앞으로 정말 많이 하게 될 텐데, 대부분의 경우 이번 처럼 다른 로직을 필요로 하지 않고 매핑할 컴포넌트만 반환해준답니당 

이럴 땐 중괄호와 return문을 생략하고 바로 `data.map((item,idx)=> <Btn>...</Btn>)` 이렇게 표현해주면 간결해지는데요, return 값만 있을 경우 return 문을 생략할 수 있는건 ES6의 화살표 함수 개념이니까 참고하세요 > <

📚 [MDN 문서](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
```suggestion
        {todoList.map((todo, index) => (
            <TodoCard
              key={index}
              todo={todo.todo}
              isdone={todo.isdone}
              deleteTodo={deleteTodo}
              toggleTodo={toggleTodo}
            />
        ))}
```